mozilla/kuma
# fix bug 797911 - update kumascript submodule

https://bugzilla.mozilla.org/show_bug.cgi?id=797911

To test:
1. git submodule update --init
2. restart kumascript (?)
3. Add some kumascript with unicode escape characters:

```
{{ languages( { "de": "de/Entwicklerhandbuch/Build_Anweisungen/Windows_Build_Voraussetzungen", "es": "es/Requerimientos_para_la_compilaci\u00f3n_en_Windows", "fr": "fr/Pr\u00e9alables_\u00e0_la_compilation_sous_Windows", "ja": "ja/Windows_Build_Prerequisites", "zh-cn": "cn/Windows_Build_Prerequisites" } ) }}
```
1. Force-refresh to force macro evaluation

Should not see any more errors.
